src/patternfly/components/AppLauncher/app-launcher-menu-item-external-icon.hbs

```diff
@@ -1,6 +1,13 @@
-<i class="pf-c-app-launcher__menu-item-external-icon fas fa-external-link-alt{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
-  aria-hidden="true"
+<span class="pf-c-app-launcher__menu-item-external-icon
+  {{#if app-launcher-menu-item-external-icon--modifier}} 
```

Comment:
Currently this generates the markup `<span class="pf-c-app-launcher__menu-item-external-icon ">...` if there is nothing passted to `app-launcher-menu-item-external-icon--modifier`. Moving `{{#if ...` up a line so that it is `<span class="pf-c-app-launcher__menu-item-external-icon{{#if ...` is how we usually write it, so as to be consistent with the rest of our examples and to remove any extra space characters in the `class` attribute.